.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.7
+2.7.7
```

Comment:
Please move this change to `.ruby-version` to a different PR, unless it is somehow related to aggregate queries or is required for this PR to pass CI.

Reply:
> Please move this change to `.ruby-version` to a different PR, unless it is somehow related to aggregate queries or is required for this PR to pass CI.

Bump

Reply:
Hmm. It was showing up in my diff. I probably had the wrong diff selected. Sorry about that. It's clearly done now.